packages/algob/test/lib/ssc.ts

```diff
@@ -11,20 +13,113 @@ describe("Convert integer to big endian", () => {
    * fmt.Println(buf)
    */
   it("should return correct big endian for given integers", () => {
-    let res = intToBigEndian(20);
-    let expected = new Uint8Array([0, 0, 0, 0, 0, 0, 0, 20]);
+    let res = uint64ToBigEndian(0);
+    let expected = new Uint8Array([0, 0, 0, 0, 0, 0, 0, 0]);
     assert.deepEqual(res, expected);
 
-    res = intToBigEndian(700000000);
+    res = uint64ToBigEndian(20);
+    expected = new Uint8Array([0, 0, 0, 0, 0, 0, 0, 20]);
+    assert.deepEqual(res, expected);
+
+    res = uint64ToBigEndian(700000000);
     expected = new Uint8Array([0, 0, 0, 0, 41, 185, 39, 0]);
     assert.deepEqual(res, expected);
 
-    res = intToBigEndian(233654);
+    res = uint64ToBigEndian(233654);
     expected = new Uint8Array([0, 0, 0, 0, 0, 3, 144, 182]);
     assert.deepEqual(res, expected);
 
-    res = intToBigEndian(9007199254740991);
+    res = uint64ToBigEndian(Number.MAX_SAFE_INTEGER);
     expected = new Uint8Array([0, 31, 255, 255, 255, 255, 255, 255]);
     assert.deepEqual(res, expected);
   });
```

Comment:
let's merge these tests with the test below or remove it(as they are same).

Reply:
In the below tests we are operating on bigint not number (to represent upto 64 bit). I don't want to remove them, will merge.